egs/voxceleb/v1/local/make_voxceleb1.pl

```diff
@@ -28,45 +28,28 @@
 closedir $dh;
 
 if (! -e "$data_base/voxceleb1_test.txt") {
-  system("wget -O $data_base/voxceleb1_test.txt http://www.openslr.org/resources/49/voxceleb1_test.txt");
-}
-
-if (! -e "$data_base/vox1_meta.csv") {
-  system("wget -O $data_base/vox1_meta.csv http://www.openslr.org/resources/49/vox1_meta.csv");
+  system("wget -O $data_base/voxceleb1_test.txt http://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test.txt");
```

Comment:
Can you point the new URL to http://www.openslr.org/resources/49/voxceleb1_test_v2.txt instead? We prefer to host these things ourselves where possible.